runtime.go

```diff
@@ -278,21 +279,28 @@ func callParamConvert(val reflect.Value, t reflect.Type) reflect.Value {
 
 	switch t.Kind() {
 	case reflect.Int, reflect.Int8, reflect.Int16, reflect.Int32, reflect.Uint, reflect.Uint8, reflect.Uint16, reflect.Uint32, reflect.Uint64, reflect.Float32, reflect.Float64:
-		if val.Kind() == t.Kind() {
```

Comment:
1. There is already this check in the function.
2. I don't want to return here, I want the section after the switch (starting from line 291) to run. This is to make custom integer types work as expected (see Test_reflectCustomType).